aws/resource_aws_ec2_capacity_reservation_test.go

```diff
@@ -88,6 +88,7 @@ func TestAccAWSEc2CapacityReservation_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "instance_match_criteria", "open"),
 					resource.TestCheckResourceAttr(resourceName, "instance_platform", "Linux/UNIX"),
 					resource.TestCheckResourceAttr(resourceName, "instance_type", "t2.micro"),
+					resource.TestCheckResourceAttrSet(resourceName, "owner_id"),
```

Comment:
Can verify that the `owner_id` is set to the current AWS account ID.

```suggestion
					testAccCheckResourceAttrAccountID(resourceName, "owner_id"),
```

Reply:
Didn't know that kind of test helper exists. Thank you!